zk_toolbox/crates/zk_inception/src/commands/ecosystem/create.rs

```diff
@@ -108,3 +111,14 @@ fn clone_era_repo(shell: &Shell) -> anyhow::Result<PathBuf> {
     .run()?;
     Ok(shell.current_dir().join("zksync-era"))
 }
+
+fn update_submodules_recursive(shell: &Shell, link_to_code: &Path) -> anyhow::Result<()> {
+    let _dir_guard = shell.push_dir(link_to_code);
+    Cmd::new(cmd!(
+        shell,
+        "git submodule update --init --recursive
+"
+    ))
+    .run()?;
+    Ok(())
+}
```

Comment:
Is this necessary? Aren't we already covered when cloning the repo?

Reply:
It's for the guys, who cloned the repo by themselves

Reply:
In that case shouldn't we also checkout the `main` branch in zksync-era?

Reply:
I don't think so. Because it will stop the development process